### DISCUSSION TITLE
Prevent accidental file deletion – always send to Trash

### DISCUSSION THREAD

**Discussion by binarylogic** [zed-industries/zed#22121] [Feature Requests]
Hi, I accidentally deleted a file today without realizing it, and it didn’t appear in my Trash. I’m not sure how this happened, but I want to prevent this in the future.

1. **Is there a way to disable permanent deletion entirely** so that files always go to Trash?
2. I’m curious—what’s the reasoning behind allowing files to be permanently deleted in the first place? I can’t think of a scenario where this would be preferred.

Thanks.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
